sagemaker-spark-sdk/src/test/scala/com/amazonaws/services/sagemaker/sparksdk/protobuf/ProtobufConverterTests.scala

```diff
@@ -144,16 +219,41 @@ class ProtobufConverterTests extends FlatSpec with MockitoSugar with BeforeAndAf
     assert(recordToShape(record) == sparseVector.size)
   }
 
+  private def validateSparse(record: Record, sparseMatrix: SparseMatrix) : Unit = {
+    print(sparseMatrix)
```

Comment:
Could you remove the print?

Reply:
Absolutely. I left this behind during some "test debugging".